contribute/documentation.rst

```diff
@@ -15,9 +15,7 @@ These rules govern *how* you should write your documentation to avoid problems w
 Title
 ^^^^^
 
-All pages must have a document title. This title is shown on the table of contents (to the left) and at the top of the page.
-
-The title, underlined with the Equals sign, is shown in the table of contents to the left of the page.
+All pages must have a document title. This title is underlined with equals signs. It is shown on the table of contents (to the left) and at the top of the page.
```

Comment:
Three main clauses in a row sound weird. Consider using commas. You see what i mean.